pkg/controller/shared_informer.go

```diff
@@ -206,7 +206,7 @@ func (f *deploymentConfigInformer) Informer() framework.SharedIndexInformer {
 		},
 		informerObj,
 		f.defaultResync,
-		cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc},
+		cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc, oscache.ImageStreamReferenceIndex: oscache.ImageStreamReferenceIndexFunc},
```

Comment:
I must have missed this in the shared cache review, please address it in either this issue or one of the coming refactors.